### PR TITLE
[ISSUE-62] Windows Compatibility

### PR DESCRIPTION
**Pull request by Rafaelbenck**
### Description:
These changes run vulkan on the Windows operating system, working with filesystems in the correct way that the system supports.

### Chore:

We will no longer need to use the open dependency of the package.json file, we will use exec for the task, as open does not execute the file as soon as the deploy is ready.

### Fixed:

- #### presets ls

image

- #### prebuild
image

- #### build
 
image

When running the deploy we have a problem with the core, as it is not able to read/find the files in the default url of the windows system, we will make some adjustments in the webpack, and in the core of the project, when completing the deploy and authentication process we found some errors that windows OS users.

### CODE REVIEW

**Comment by Rafaelbenck** [comment]
We will no longer need to use the open dependency of the package.json file, we will use exec for the task, as open does not execute the file as soon as the deploy is ready.

**Comment by Rafaelbenck** [comment]
we have an error returning the list of presets 
image

**Comment by Rafaelbenck** [comment]
After fixing the file reading errors in `getPresetList.utils.js` and `getAbsoluteLibDirPath.js` there is an error to prebuild the project in get 
image

**Comment by Rafaelbenck** [comment]
Ok now our build process is right. When starting the deploy process we found this error message:

image

It seems to me that core is not able to identify the correct Url to identify the paths in windows

File: lib/build/dispatcher/dispatcher.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
-import { join, resolve } from 'path';
+import { join, resolve, sep } from 'path';
```
Comment (jotanarciso) [inline lib/build/dispatcher/dispatcher.js:1]: "SEP'  is not being used. Can remove.

File: lib/build/dispatcher/dispatcher.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 import { readFileSync, readdirSync, existsSync } from 'fs';
 import { writeFile, rm } from 'fs/promises';
 import { Webpack } from '#bundlers';
-import { feedback, generateTimestamp, getAbsoluteLibDirPath } from '#utils';
+import { feedback, generateTimestamp, getAbsoluteLibDirPath, debug } from '#utils';
```
Comment (jotanarciso) [inline lib/build/dispatcher/dispatcher.js:5]: Debug can also be removed.

**Review by MagnunAVFAzion — APPROVED** [review]

**Review by jotanarciso — APPROVED** [review]

**Pull request merged**